src/Form/Form.test.js

```diff
@@ -84,8 +84,8 @@ describe('Test the form class reading values correctly', () => {
     <input type="password" value="" autocomplete="new-password" />
     <button type="submit">Sign up</button>
 </form>`,
-            expHasValues: true,
-            expValues: { credentials: { username: 'testUsername' } },
+            expHasValues: false,
```

Comment:
Tests reverted to match https://github.com/duckduckgo/duckduckgo-autofill/pull/702/files so I am confident we're good here.

Reply:
https://github.com/duckduckgo/duckduckgo-autofill/pull/702/files#diff-6cc15f20dfd09f594b38d6cee7b2efe4b4882efd677b3769a03734403b427aeaR87-R99